code/game/area/areas/ruins/_ruins.dm

```diff
@@ -12,10 +12,13 @@
 
 
 /area/ruin/unpowered
+	name = "\improper Powerless Ruin" //let's be honest, you shouldn't be using these names
```

Comment:
unexplored location is a fine name for all of these, this will just make random maps have weird ass names
example:
https://github.com/tgstation/tgstation/blob/8d0bd5322f1524492b4d1caa8730f8e179836de2/_maps/RandomRuins/SpaceRuins/clericden.dmm#L7

Reply:
I'm just going to add areas for all of the edge cases then.

Reply:
It's a larger task but I feel like it should be done in this PR.